package.json

```diff
@@ -67,7 +67,7 @@
     "test:ftr:server": "node scripts/functional_tests_server",
     "test:jest": "node scripts/jest",
     "test:jest_integration": "node scripts/jest_integration",
-    "typecheck": "node scripts/type_check"
+    "test:type_check": "node scripts/type_check"
```

Comment:
why did you rename the script? 
For me `test:type_check` would mean run typechecks on the test code, or similar - this is not test-related, but typechecking any code